src/FeatureSet.php

```diff
@@ -96,6 +97,11 @@ class FeatureSet
     private $timeGenerator;
 
     /**
+     * @var ValidatorInterface
+     */
+    private $validator;
```

Comment:
I already removed this from the `UuidFactory` since it had nothing to do with creating UUIDs. Should I remove it from here as well since it's dead use unless some interfaces with `FeatureSet` directly to get/set the validator?